### DISCUSSION TITLE
The connection is not allowed. The license has expired or is invalid.

### DISCUSSION THREAD

**Discussion by yaroz** [rustdesk/rustdesk#5842] [Q&A] [answered]
Receiving this message on one of our servers, but all of the rest seem to be connecting fine.  We have the pro license, so I restarted the docker container.  I also updated the software to the latest nightly build on the server that we're having a problem connecting with.  Is there anything else I should look at?

**Comment by dinger1986** [comment]
have you setup docker using host? or followed the docs https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/installscript/docker/

You can release your license to use again if it sees this a new machine https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/license/#invoices-license-retrieval-and-migration

**Reply by yaroz** [reply]
Yes, I set up using the docker-compose.yml example.  I just verified it was set up as host.

I did release and relicense the server, and now we're getting "Failed to connect via rendezvous server: Please try later"  This server is on a different subnet than the other servers, but it's been working for weeks now and was working this morning.  Any ideas on what to check next?

**Reply by rustdesk** [reply]
Some similar: 
https://github.com/rustdesk/rustdesk/issues/4267
https://github.com/rustdesk/rustdesk/discussions/5817

**Comment by yaroz** [comment]
I ended up setting up the relay on the admin portal, and setting the Always Use Relay, and that seems to have worked.  Before I did this, I didn't see anything in the logs for hbbr.  When I did this, I saw this in the hbbs logs `INFO [src/api.rs:1910] ALWAYS_USE_RELAY updated to TRUE`, then I restarted and I wasn't able to connect again.  I did a "docker logs hbbs" and noticed `INFO [src/rendezvous_server.rs:277] ALWAYS_USE_RELAY=N`.  I refreshed the webportal and the switch was still on, so I turned it off and saw `INFO [src/api.rs:1910] ALWAYS_USE_RELAY updated to FALSE`, then turned it back on, and saw ` INFO [src/api.rs:1910] ALWAYS_USE_RELAY updated to TRUE` and I could connect again.  Is this setting not persistent?

**Reply by rustdesk** [reply, marked as answer]
> Is this setting not persistent?

Thanks, you find us a bug. Just fixed in https://github.com/rustdesk/rustdesk-server-pro/releases/tag/1.1.10-3, you can try out.

**Reply by yaroz** [reply]
Perfect.. I just replaced latest with that tag, pulled the new image and tested it out.. it worked.  Thank you.

**Comment by parbzip** [comment]
VER 1.4.3
The connection is not allowed. The license has expired or is invalid.

https://github.com/user-attachments/assets/7cc11974-913f-42e6-84f5-631c5af0428f

**Reply by yaroz** [reply]
@parbzip Has your license expired?
